apps/cowswap-frontend/src/common/pure/OrderProgressBarV2/container/StepsWrapper.tsx

```
@@ -0,0 +1,79 @@
+    }
+  }, [currentStep, steps.length])
+
+  return (
+    <styledEl.StepsContainer
+      $height={containerHeight}
+      $minHeight={isCancelling ? '80px' : undefined}
+      bottomGradient={!isCancelling}
+    >
+      <styledEl.StepsWrapper ref={wrapperRef}>
+        {steps.map((step, index) => {
+          const customTitle = customStepTitles && customStepTitles[index]
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)